feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/presentation/handling/SendingNotificationHandler.kt

```diff
@@ -0,0 +1,35 @@
+package io.novafoundation.nova.feature_push_notifications.data.presentation.handling
+
+import android.content.Context
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.google.firebase.messaging.RemoteMessage
+import io.novafoundation.nova.common.resources.ResourceManager
+import io.novafoundation.nova.feature_push_notifications.R
+
+class SendingNotificationHandler(
+    private val context: Context,
+    notificationManager: NotificationManagerCompat,
+    resourceManager: ResourceManager,
+) : BaseNotificationHandler(
+    notificationManager,
+    resourceManager,
+    R.string.default_notification_channel_id,
+    R.string.default_notification_channel_name,
+) {
+
+    override fun handleNotificationInternal(notificationManager: NotificationManagerCompat, channelId: String, message: RemoteMessage): Boolean {
+        // TODO check if the notification has a send type
+
+        val notification = NotificationCompat.Builder(context, channelId)
+            .setContentTitle("\uD83D\uDCB8 Sent [Account name/id]") // TODO stub
+            .setContentText("Sent 0.5 ETH (\$900) to 0xc6c...81cdd on Moonbeam") // TODO stub
```

Comment:
Pending TODOs. I suggest to mark future TODO with the task number if possible so during review we can check if it is a forgotten todo or the inentional one

Reply:
Okay, I'll do this in the next time!
But in this case I'm sure we will not forget about this since we still need to handle other push types and I'll return to this task anyway